src/json-typing.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "*.json" {
+    const value: any;
+    export default value;
+  }
+  
```

Comment:
I didn't know this file purpose, I understand it is to read use jsons.
See my comment in the component ts file and this file will probably not be necessary.

Reply:
Yes, It is need to get the json file, since we will use the backend.service we can remove it